openpype/settings/entities/schemas/projects_schema/schema_project_houdini.json

```diff
@@ -19,6 +19,39 @@
                 {
                     "type": "template",
                     "name": "template_host_color_management_ocio"
+                },
+                {
+                    "key": "workfile",
+                    "type": "dict",
+                    "label": "Workfile",
+                    "collapsible": true,
+                    "checkbox_key": "enabled",
+                    "children": [
+                        {
+                            "type": "boolean",
+                            "key": "enabled",
+                            "label": "Enabled"
+                        },
+                        {
+                            "type": "label",
+                            "label": "Empty values will be skipped, allowing any existing env vars to pass through as defined.\nNote: The render space in Houdini is always set to the 'scene_linear' role."
+                        },
+                        {
+                            "type": "text",
+                            "key": "default_display",
+                            "label": "Display"
+                        },
+                        {
+                            "type": "text",
+                            "key": "default_view",
+                            "label": "View"
```

Comment:
Good idea to have a default list of active views. Is there any particular reason we are not using list type schema. Also `Default active views` seems to be a better label.

Reply:
`Default active views` label is far better!

>  there any particular reason we are not using list type schema?

as far as I know, admins should enter a colon `:` separated list.
having one text input field
- is pretty straight forward and easier to maintain and debug. 
- mirrors the usual workflow where admins are used to do something like 
    ```
    set OCIO_ACTIVE_VIEWS=DCDM:sRGB
    ```
@fabiaserra  or @BigRoy  could you tell us what do you see from your perspective ?

Reply:
I personally don't see a reason to make it a list - we can just stay close to how the OCIO env var behaves. I'd try and avoid wrapping too much stuff around what already has functionality. In essence we're just setting an env var, let's keep it as light weight as that.